gladier_tools/posix/encrypt.py

```diff
@@ -0,0 +1,52 @@
+from gladier import GladierBaseTool, GladierBaseClient, generate_flow_definition
+from pprint import pprint
+
+
+def encrypt(**data):
+    import os
+    from cryptography.fernet import Fernet
+    from cryptography.hazmat.primitives import hashes
+    from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+    import base64
+
+    password = bytes(data['key'], 'utf-8')
+    salt= b'\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00'
+    # salt = os.urandom(16)
```

Comment:
Why is the salt here set to all zeros? Dose the os.urandom()` line work?

Reply:
The os.urandom line works, but it uses the input key and a random salt to create the encryption key, which would then have to be stored in order to decrypt the file. I used all 0s as a temporary fix so that the same key can be used to decrypt the encrypted file